### DISCUSSION TITLE
Turn off repo-actions on a page

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#1066] [Q&A] [answered]
Recent update (e.g. v0.9.515) seems to show repo-actions on about pages too.
Is it possible to turn off repo-actions only on certain pages because it breaks the symmetrical layout?

![image](https://user-images.githubusercontent.com/50911393/172032620-1e3b171f-eb28-43b5-90a2-6d72f980c3c9.png)

Source code is here. https://github.com/eitsupi/rocker-website/tree/976b4ba897ef3fddd8cbe4266d3f8c1900d8c5fb

**Comment by eitsupi** [comment, marked as answer]
It seems related to the change for solving #691

**Reply by dragonstyle** [reply]
Yep! I will fix for the about page- sorry about that!
